src/lib/Scenes/Show/Screens/Detail.tsx

```diff
@@ -198,15 +198,6 @@ export const DetailContainer = createFragmentContainer(
       description
       city
       is_local_discovery
-      location {
```

Comment:
removed since we already had https://github.com/artsy/emission/pull/1478/files#diff-bc10a37d56d39bee898f2d52abfcbcdfR209 few lines further down, correct me if im wrong but this seemed duplicate.

Reply:
Not entirely. When using proper Relay containers you should _not_ worry about other fragments fetching the same data, as doing so would couple your components. Relay goes out of its way to enforce this through ‘data masking’ and will dedupe those selections in the final compiled query.

Required reading https://facebook.github.io/relay/docs/en/thinking-in-relay.html#data-masking

Reply:
Most importantly, this file’s component seems to only care about the existence of a location, not the actual contents. For the existence check to work, only the stuff you left in is needed.

```graphql
      location {
        ...LocationMap_location
      }
```